src/tribler/core/utilities/tests/test_async_interuption.py

```diff
@@ -0,0 +1,47 @@
+import asyncio
+import os
+import threading
+
+import pytest
+
+from tribler.core.utilities.async_interruption import interruption
+
+
+@pytest.mark.looptime(False)
+async def test_without_interruption():
```

Comment:
It is strange that the added test lines are not covered by tests; it looks like pytest does not run this test file

Reply:
I'll check it.

Reply:
Fixed